sql/branch.sql

```diff
@@ -0,0 +1,4 @@
+BEGIN;
+    UPDATE payment_instructions SET due = 0 WHERE amount = 0 AND due != 0;
+    UPDATE payment_instructions SET due = floor(9.81/(amount)) * amount WHERE due > 9.81;
```

Comment:
P.S. If you replace `master` in that URL with the [current version number](https://gratipay.com/version.txt) then you get a permalink—master is likely to drift over time. You can also hit "y" as a GitHub keyboard shortcut but our version number is neater than the git hash. ;)